src/test/scala/za/co/absa/hyperdrive/trigger/models/tables/SearchableTableQueryTest.scala

```diff
@@ -52,10 +52,10 @@ class SearchableTableQueryTest extends FlatSpec with Matchers with BeforeAndAfte
       ContainsFilterAttributes(field = stringField, value = "value"),
       ContainsFilterAttributes(field = stringField2, value = "str")
     ))
-    val intRangeFilterSeq = Some(Seq(IntRangeFilterAttributes(field = longField, start = 0, end = 1)))
+    val intRangeFilterSeq = Some(Seq(IntRangeFilterAttributes(field = longField, start = Option(0), end = Option(1))))
```

Comment:
Why do you use `Option` instead of `Some`? It's shorter :) But no need to change ofc

Reply:
In this case it doesnt matter. It handles nulls from java better.